src/NuGet.Jobs.RegistrationComparer/HiveComparer.cs

```diff
@@ -159,7 +159,7 @@ public class HiveComparer
                     }
                     catch (InvalidOperationException ex)
                     {
-                        ResultWriter.WriteWarning(ex.Message);
+                        _logger.LogWarning(ex.Message);
```

Comment:
Should we log the full warning given that we're no longer constrained by a file?

Reply:
Sure, seems fine